config.json

```diff
@@ -19,7 +19,16 @@
                     "name": "SDK Configuration"
                 },
                 {
-                    "name": "Integration Basics"
+                    "name": "Integration Basics iOS",
+                    "platforms": ["iOS"]
+                },
+                {
+                    "name": "Integration Basics Android",
+                    "platforms": ["android"]
+                },
+                {
+                    "name": "Integration Basics Java",
+                    "platforms": ["javaSE, javaEE"]
```

Comment:
`Getting Started / Install` uses one file with sperate platform tags. 
This PR creates 3 files. 
Should we leave this as if for now or move it until one file?